package.json

```diff
@@ -24,24 +24,24 @@
         "@babel/core": "7.22.11",
         "@cypress/vite-dev-server": "4.0.1",
         "@testing-library/react": "14.0.0",
-        "@types/node": "18.7.21",
-        "@types/react": "18.0.35",
-        "@types/react-dom": "18.2.1",
+        "@types/node": "18.17.12",
+        "@types/react": "18.2.21",
+        "@types/react-dom": "18.2.7",
         "@vitejs/plugin-react": "2.2.0",
         "cypress": "12.5.1",
-        "cypress-real-events": "1.7.6",
-        "happy-dom": "7.6.7",
+        "cypress-real-events": "1.10.1",
+        "happy-dom": "10.11.2",
         "msw": "0.48.1",
-        "postcss": "8.4.28",
-        "tailwindcss": "3.2.7",
+        "postcss": "8.4.29",
+        "tailwindcss": "3.3.3",
         "ts-node": "10.9.1",
-        "typescript": "4.8.4",
+        "typescript": "4.9.5",
         "vite": "3.2.7",
-        "vitest": "0.25.0"
+        "vitest": "0.34.3"
     },
     "dependencies": {
-        "@frontify/fondue": "12.0.0-beta.309",
-        "glob": "10.0.0",
+        "@frontify/fondue": "12.0.0-beta.316",
+        "glob": "10.3.4",
         "react": "18.2.0",
         "react-dom": "18.2.0"
```

Comment:
Could you change the deps (in all packages) to always be `^version` except for Fondue?

Reply:
Added everywhere 👍